src/services/ogc-features/src/main/resources/ogc-features/1.0.1/api.yaml

```diff
@@ -334,6 +335,15 @@ components:
         single:
           value: test
           summary: testtt
+    crs:
+        name: crs
+        in: query
+        description: |-
+            The coordinate reference system of the coordinates in the spatial extent (property `bbox`). The default reference system is WGS 84 longitude/latitude.
+            In the Core this is the only supported coordinate reference system. Extensions may support additional coordinate reference systems and add additional enum values.
+        required: false
+        schema:
+            type: string
```

Comment:
I would expect that the openapi allows to provide a default value to be set, for not mandatory parameters.